README.md

```diff
@@ -27,18 +118,26 @@ You can include the token when performing actions that require it:
 ```
 
 ## Configuration
+If SMD and/or BSS are configured to use cacerts and JWTs you can specify these with
+```bash
+./ochami-cli --ca-cert --access-token ...
+```
+you can also set environemnt variables for BOTH of these:
+```bash
+export CACERT=/path/to/cacert
+export ACCESS_TOKEN=eyJhbGciOiJ
+```
 
-Here is an example `config.yaml` to get started using the `ochami-cli` tool:
-
+Other helpful environment variables:
+Skip having to specify `--smd-url`
 ```bash
-nodes:
-  - name: cg01
-    xname: x1000c1s7b0n0
-    mac: b4:...
-    ipaddr: 172.16.0.1
-  - name: cg02
-    xname: x1000c1s7b1n0
-    mac: b4:...
-    ipaddr: 172.16.0.2
-access-token: eyJhbGciOiJ...
-```
\ No newline at end of file
+export SMD_URL=http://smd:27779/hsm/v2
+```
+Skip having to specify `--bss-url`
+```bash
+export BSS_URL=http://bss:27778/boot/v1
+```
```

Comment:
I think it might make sense to put `BSS_URL` and `SMD_URL` in the corresponding BSS and SMD sections above where it discusses the corresponding flags (i.e. `--bss-url` and `--smd-url`) since they do the same thing.

Reply:
6bc923f